api/src/org/apache/cloudstack/api/response/AutoScaleVmProfileResponse.java

```diff
@@ -75,6 +75,7 @@
     @Parameter(name = ApiConstants.CS_URL,
                type = CommandType.STRING,
                description = "the API URL including port of the CloudStack Management Server example: http://server.cloud.com:8080/client/api?")
+    // leaving cloud.com reference above as it serves only as an example
```

Comment:
This examples with URL, what about using notations like: 
http://<apacheCloudStackServer>:[port]/client/api?

I think administrators/operators are familiar with those notations